ingest-app/docs/api.md

```diff
@@ -22,6 +22,9 @@ Join the [CMR Client Developer Forum](https://wiki.earthdata.nasa.gov/display/CM
   * /providers/\<provider-id>/services/\<native-id>
     * [PUT - Create or update a service.](#create-update-service)
     * [DELETE - Delete a service.](#delete-service)
+  * /providers/\<provider-id>/tools/\<native-id>
```

Comment:
minor question: should the provider types be alphabetical or is there some other logic to their order?

Reply:
I'm not sure what you meant. it's provider-id, not provider types. there is no ordering involved. You need to pass in a specific provider-id, like DEMO_PROV, when you create/update/delete a tool.